Jenkinsfile

```diff
@@ -97,8 +98,6 @@ void withMavenEnv(List envVars = [], def body) {
     }
 }
 
-// This hacky method is used because File is not whitelisted,
-// so we can't use renameTo or friends
```

Comment:
You could not use them anyway since the workspace is on an agent. This comment is just misleading.